src/services/actions/addTodo.js

```diff
@@ -1,7 +1,12 @@
-import { ADD } from "../constants/constants"
+// import { ADD } from "../constants/constants"
```

Comment:
Bhai we may remove the line if not needed.

Reply:
removed